docs/using/wallets.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Wallets
+authors: "Tim McMackin"
+last_update:
+  date: 30 December 2024
+---
+
+Wallets are tools that manage accounts on blockchains like Tezos.
+Their primary purpose is to store an account's private key securely and encrypt (or "sign") transactions with that key without exposing the key.
+For more information on accounts, see [Accounts](/using/accounts).
+
+Some wallets can do other tasks, including:
+
+- Showing the tokens that an account holds
+- Providing information on the value of tokens
+- Buying, selling, and exchanging tokens
+- Showing the transaction history of an account
+
```

Comment:
Other things supported:
-  batching! We should highlight this as we don't use it enough

Reply:
Added "Batching multiple transactions to save on transaction fees"